Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+/// Error codes associated with PayPal.
+enum BTPayPalError: Error, CustomNSError, LocalizedError {
```

Comment:
We created a number of new, specific error cases, but many of them aren't tested yet. We've created a ticket (DTBTSDK-2190) to add test coverage.